### DISCUSSION TITLE
Jedis with Redis 7.0

### DISCUSSION THREAD

**Discussion by vineelyalamarthy** [xetorthio/jedis#3276] [Q&A]
It seems Redis 7.0 is released last year. Does Jedis work with Redis 7.0 as well. Asking because its a major version change from Redis. 

And if we have our apps using Jedis and working with 6.2.6 would we potentially need any changes.

**Comment by sazzad16** [comment]
> Does Jedis work with Redis 7.0 as well.

Yes.

> if we have our apps using Jedis and working with 6.2.6 would we potentially need any changes.

If you're using STRALGO LCS command (`strAlgoLcs` method), remove it. I think that's all.
